services/apps/alcs/src/alcs/staff-journal/staff-journal.dto.ts

```diff
@@ -40,14 +42,16 @@ export class CreateNoticeOfIntentStaffJournalDto {
   body: string;
```

Comment:
Did you mean to remove `body` after extending base?

Reply:
Yes, fixed